kit/brain_board.md

```diff
@@ -9,3 +9,26 @@ Brain Board
 <img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
 The Odroid U3+ Brain Board provided with your kit is what runs the code you write
 and controls the other peripheral boards.
+
+It can be controlled remotely over [WiFi](/docs/kit/wifi).
+
+### Flashing SD card
+
+To fully update your Brain Board's software, or refresh it if you think it's not working correctly, you can flash our SD card image onto the microSD card in your Brain Board.
+
+To update the SD card, you'll need to download our image, `robot-image.img.xz`. The flashing procedure is identical to flashing Raspberry Pi images.
```

Comment:
Question: Is this definitely the name of the file?

Reply:
As we haven't published an image yet, no, but we can change this on release.

Reply:
The image names will vary depending on the build number obviously.

We can basically call it whatever we want though.